docs/news/posts/2024-01-22.md

```diff
@@ -0,0 +1,10 @@
+---
+date: 2024-01-22
+---
+
+# Wulver Maintenance
+
+!!! important "Wulver Monthly Maintenance" 
+
+
+Wulver will be out of service for maintenance once a month for updates, repairs, and upgrades.  The schedule is 9 a.m. to 9 p.m. the second Tuesday of every month.  During the maintenance period, all jobs and logins will be terminated.
```

Comment:
Jobs will not be terminated. Instead jobs that do not end before the maintenance window begins will be held until the maintenance is complete and the cluster is returned to production.